### PR TITLE
Fix #1094:Added confirmation for Loan Submission

### LINKED ISSUE #1094: Confirmation after Submitting Loan Application
**Summary:**

After submitting a loan application in Apply for Loan, the user should be asked for confirmation.

**Steps to reproduce:**

Apply for Loan->Submit Loan

**Expected behaviour:**

Confirmation tab should be opened.

**Observed behaviour:**

No confirmation.

**Device and Android version:**

Redmi Note 3

### PR DESCRIPTION
**Pull request by nikhil-sachdeva**
Fixes #1094

- [x] Apply the `AndroidStyle.xml` style template to your code in Android Studio.

- [x] Run the unit tests with `./gradlew check` to make sure you didn't break anything

- [x] If you have multiple commits please combine them into one commit by squashing them.

### CODE REVIEW

File: app/src/main/java/org/mifos/mobilebanking/ui/fragments/LoanApplicationFragment.java [hunk 1]
```diff
@@ -218,11 +220,34 @@ void onSubmitLoanApplication() {
...
-        } else {
-            submitUpdateLoanApplication();
-        }
+        confirmLoanSubmission();
+    }
+
+    private void confirmLoanSubmission() {
+        new MaterialDialog.Builder()
+                .init(getContext())
+                .setMessage("Are you sure you want to submit the loan application?")
```
Comment (shiv07tiwari) [inline app/src/main/java/org/mifos/mobilebanking/ui/fragments/LoanApplicationFragment.java]: This string should not be hardcoded.

**Review by shiv07tiwari — COMMENTED** [review]
https://github.com/openMF/mifos-mobile/pull/1121/files#r266428959

File: app/src/main/java/org/mifos/mobilebanking/ui/fragments/LoanApplicationFragment.java (hunk 1 above)
Comment (nikhil-sachdeva) [reply app/src/main/java/org/mifos/mobilebanking/ui/fragments/LoanApplicationFragment.java]: Yes, I'm updating the PR.

**Comment by prashantkh19** [comment]
@nikhil-sachdeva can you resolve the merge conflicts?

**Review by prashantkh19 — APPROVED** [review]

**Comment by prashantkh19** [comment]
Closing this as #1187 solves the issue.

**Pull request closed without merging**